tpmutil/tbs/tbs_windows.go

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2018, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tbs
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#package-comments

Reply:
I added tbs.go file with the package description. This file will also make sure the package builds on Linux (since the *_windows.go files will not appear).

Reply:
I think tbs.go makes things a bit more confusing.
It just looks like an empty package on linux. If you remove it, builds on linux will still pass because it won't see anything in this directory.

Reply:
Okay. Sounds good!